docs/source/community/persons_of_interest.rst

```diff
@@ -344,17 +380,53 @@ TorchAudio
 TorchRec
 ~~~~~~~~
 
--  Dmytro Ivchenko (`divchenko <https://github.com/divchenko>`__)
 -  Colin Taylor (`colin2328 <https://github.com/colin2328>`__)
+-  Paul Zhang (`PaulZhang12 <https://github.com/PaulZhang12>`__)
+-  (emeritus) Dmytro Ivchenko (`divchenko <https://github.com/divchenko>`__)
 
 TorchX
 ~~~~~~
 
--  Tristan Rice (`d4l3k <https://github.com/d4l3k>`__)
--  Kiuk Chung (`kiukchung <https://github.com/kiukchung>`__)
+-  (emeritus) Tristan Rice (`d4l3k <https://github.com/d4l3k>`__)
+-  (emeritus) Kiuk Chung (`kiukchung <https://github.com/kiukchung>`__)
 
-TorchData / TorchArrow
+TorchData
 ~~~~~~~~~~~~~~~~~~~~~~
 
--  Wenlei Xie (`wenleix <https://github.com/wenleix>`__)
+-  Andrew Ho (`andrewkho <https://github.com/andrewkho>`__)
+-  Divyansh Khanna (`divyanshk <https://github.com/divyanshk>`__)
+
+TorchArrow
+~~~~~~~~~~~~~~~~~~~~~~
+
+-  (emeritus) Wenlei Xie (`wenleix <https://github.com/wenleix>`__)
 -  (emeritus) Vitaly Fedyunin (`VitalyFedyunin <https://github.com/VitalyFedyunin>`__)
+
+ExecuTorch (Edge, Mobile)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  Mergen Nachin (`mergennachin <https://github.com/mergennachin>`__)
+-  Kimish Patel (`kimishpatel <https://github.com/kimishpatel>`__)
+-  Dave Bort (`dbort <https://github.com/dbort>`__)
+-  Martin Yuan (`iseeyuan <https://github.com/iseeyuan>`__)
+
+TorchTune
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  Kartikay Khandelwal (`kartikayk <https://github.com/kartikayk>`__)
+-  Evan Smothers (`ebsmothers <https://github.com/ebsmothers>`__)
+-  Joe Cummings (`joecummings <https://github.com/joecummings>`__)
+
+TorchChat
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  Jack Khuu (Jack-Khuu)
```

Comment:
@albanD nit, but can you make this a link? Thanks

Reply:
Ho my regex to make everything a link did not expect special characters in the github id :D 
Fixing!